### PR TITLE
doc(k8s): reorg README and update with the new deployment method #89

### PR DESCRIPTION
**Pull request by tadayosi**
- How to create a ServiceAccount for k8s deployment will be added later.

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,220 +1,174 @@
...
-  dc/hawtio-online deploys istag/hawtio-online:latest 
-    deployment #1 deployed 2 minutes ago - 1 pod
-```
+### Kubernetes
 
-Open the route URL displayed above from your Web browser to access the Hawtio Online console.
+#### Proxying certificates
 
-## OpenShift 4
+For Kubernetes, proxing certificates are not supported, so you don't need to go through the steps.
```
Comment (astefanutti) [inline README.md:48]: Should we rather say it's disabled by default. It's possible to use a proxying client certificate, even for Kubernetes, the may difference with OpenShift is that a custom CA must be generated / used, and mounted / configured into the Jolokia agent.

**Review by astefanutti — APPROVED** [review]
Great!

File: README.md (hunk 1 above)
Comment (tadayosi) [reply README.md:48]: Thanks. Yes, you are right. I'll change that later.

Comment (astefanutti) [reply README.md:48]: Agreed, that can be improvement in a subsequent PR.

**Pull request merged**